### PR TITLE
Actualize UUV Parameters and UUV Plugin

### PR DESCRIPTION
**Pull request by NRottmann**
With this commit I actualize the parameter for the UUV HippoCampus from the Technical University Hamburg-Harburg. The parameters have now been determined by experiments.

### CODE REVIEW

File: include/gazebo_uuv_plugin.h [hunk 1]
```diff
@@ -67,6 +67,8 @@ class GazeboUUVPlugin : public ModelPlugin {
     void CommandCallback(CommandMotorSpeedPtr &command);
     double command_[4];
 
+    double absolute(double x);
```
Comment (TSC21) [inline include/gazebo_uuv_plugin.h]: you don't need this. You can simply use `std::abs( double arg )`.

File: src/gazebo_uuv_plugin.cpp [hunk 2]
```diff
@@ -115,6 +115,17 @@ void GazeboUUVPlugin::CommandCallback(CommandMotorSpeedPtr &command) {
 
 }
 
+// gives back absolute value of a double number
```
Comment (TSC21) [inline src/gazebo_uuv_plugin.cpp]: no need.

File: src/gazebo_uuv_plugin.cpp [hunk 3]
```diff
@@ -131,7 +142,7 @@ void GazeboUUVPlugin::OnUpdate(const common::UpdateInfo& _info) {
   for(int i = 0; i < 4; i++) {
     
     // Currently a rotor index hack to get over IMU link being first, since rotor_links_[0] would be the IMU
-    math::Vector3 rotor_force(motor_force_constant_ * command_[i], 0, 0);
+    math::Vector3 rotor_force(motor_force_constant_ * command_[i] * absolute(command_[i]), 0, 0);
```
Comment (TSC21) [inline src/gazebo_uuv_plugin.cpp]: `math::Vector3 rotor_force(motor_force_constant_ * command_[i] * std::abs(command_[i]), 0, 0);`

File: src/gazebo_uuv_plugin.cpp [hunk 4]
```diff
@@ -141,7 +152,7 @@ void GazeboUUVPlugin::OnUpdate(const common::UpdateInfo& _info) {
     // directly to main body X axis
     int propeller_direction = ((i+1)%2==0)?1:-1;            // ternary operator:  (condition) ? (if_true) : (if_false)
     math::Vector3 rotor_torque(
-      propeller_direction * motor_torque_constant_ * command_[i], 0, 0);
+      propeller_direction * motor_torque_constant_ * command_[i] * absolute(command_[i]), 0, 0);
```
Comment (TSC21) [inline src/gazebo_uuv_plugin.cpp]: `propeller_direction * motor_torque_constant_ * command_[i] * std::abs(command_[i]), 0, 0);`

**Review by TSC21 — CHANGES_REQUESTED** [review]
Just remove the `absolute` method and replace with `std::abs()` and everything is good.

**Comment by NRottmann** [comment]
Changes done

**Review by TSC21 — APPROVED** [review]

**Comment by TSC21** [comment]
Thanks!

**Pull request merged**